websites/T/The Odin Project/presence.ts

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable unicorn/prefer-dom-node-text-content */
+const presence = new Presence({
+		//The client ID of the Application created at https://discordapp.com/developers/applications
```

Comment:
Please remove this comment, it is used for the docs as a guide, but shouldn't be published.